cmd/operator-sdk/test/local.go

```diff
@@ -117,7 +117,13 @@ func testLocalGoFunc(cmd *cobra.Command, args []string) error {
 	}
 
 	if tlConfig.upLocal && tlConfig.namespace == "" {
-		return fmt.Errorf("must specify a namespace to run in when -up-local flag is set")
+		// valid case: WATCH_NAMESPACE="" in --up-local
+		// so that operator can watch All-Namespaces in localRunMode
+		// do nothing if namespace flag is explicitly set to "" and -no-setup=true
+		// return error if namespace flag is not explicitly set to "" with -no-setup flag
```

Comment:
I would suggest improve the command by doing something as:

// Allows namespace == "" in order to support cluster-scoped operators. 
// The, it will allow users to execute the command operator-sdk test local --up-local with the flag --namespace="".
// More Info: <add link for the cluster scope doc>

Reply:
done